src/plugins/LADTable/components/LADRow.vue

```diff
@@ -26,7 +26,7 @@
     class="js-lad-table__body__row"
     @contextmenu.prevent="showContextMenu"
 >
-    <td class="js-first-data">{{ name }}</td>
+    <td class="js-first-data">{{ domainObject.name }}</td>
```

Comment:
`domainObject` provided by view API is now automatically kept in sync, so I can just use Vue reactivity to update the name when it changes. No need for any event listeners at all.